RecoCTPPS/PixelLocal/src/RPixRoadFinder.cc

```diff
@@ -92,17 +92,17 @@ void RPixRoadFinder::findPattern() {
 
     it_gh2 = it_gh1;
 
-    CLHEP::Hep3Vector currPoint = it_gh1->globalPoint;
+    const auto currPoint = it_gh1->globalPoint;
     CTPPSPixelDetId currDet = CTPPSPixelDetId(it_gh1->detId);
 
     while (it_gh2 != temp_all_hits.end()) {
       bool same_pot = false;
       CTPPSPixelDetId tmpGh2Id = CTPPSPixelDetId(it_gh2->detId);
       if (currDet.rpId() == tmpGh2Id.rpId())
         same_pot = true;
-      CLHEP::Hep3Vector subtraction = currPoint - it_gh2->globalPoint;
+      const auto subtraction = currPoint - it_gh2->globalPoint;
 
-      if (subtraction.perp() < roadRadius_ && same_pot) {  /// 1mm
+      if (subtraction.Rho() < roadRadius_ && same_pot) {  /// 1mm
```

Comment:
```perp()``` and ```Rho()``` are not the same: are you fixing a bug here?

Reply:
Despite the confusing names, perp() in CLHEP and Rho() in ROOT do the same, see e.g. https://root.cern.ch/root/html608/classROOT_1_1Math_1_1DisplacementVector3D.html#a92c6c1ddbb0e06be2aa87e2cc0eb0ddb
I've verified it by explicit print outs.